### ISSUE TITLE
Description_image: caption text is necessary

### ISSUE THREAD

**Issue by AnneliesVerroken** [micado-eu/migrant_application#131]
To meet the requirements of the WCAG guidelines, an image in a text must always be accompanied by a caption text.
Currently, this is only possible under the image, in the same font size/ line spacing as all other text. This is not a desirable outcome. 
Is it possible to have a possibility to add a caption text to a specific image, that would appear right under the image, preferably in a slightly smaller font and in grey?
This goes for all text fields where you can add an image (description of guided processes, steps, event, information center, ... )

**Comment by ivchicano** [comment]
W3 indicates that the alternative text should be used to comply with WCAG: https://www.w3.org/WAI/tutorials/images/informative/
This text does not show as a caption, but it can be added if necessary.

**Comment by AnneliesVerroken** [comment]
Hey Ivan,

Yes, that is exactly what I read.
What do you suggest precisely? To do it as I did here<https://migrants.micadoproject.eu/events/62>?
So write it in the normal text edit under the picture?
I was wondering if there was a way to define it more as caption text, closer to the Image, or is that not desirable?

In general (just out of personal interest)  what level of the WCAG do we aim at? AA or AAA?

Have a nice day!


Van: Iván Chicano [mailto:notifications@github.com]
Verzonden: maandag 1 februari 2021 9:38
Aan: micado-eu/migrant_application
CC: Annelies Verroken; Author
Onderwerp: Re: [micado-eu/migrant_application] Description_image: caption text is necessary (#131)


W3 indicates that the alternative text should be used to comply with WCAG: https://www.w3.org/WAI/tutorials/images/informative/
This text does not show as a caption, but it can be added as a caption under the image if necessary.

**Comment by ivchicano** [comment]
The user will have in the image dialog a field to add a text alternative to the image such as this:
![chrome_VFUBEFTvnd](https://user-images.githubusercontent.com/46921761/106436383-503b9c80-6474-11eb-8747-0d3b72d6695f.png)
When uploading the image only the image will be shown to normal readers, but it will be helpful to people that use screen readers (it would be read aloud like "image, text alternative to the image").
This field is not mandatory if the image has only decorative purposes or there is enough text context below the image.
About the level of WCAG I'm not entirely sure, maybe @gioppoluca knows more. I personally think that level AA is reasonable enough unless it was already specified to use AAA.

**Comment by AnneliesVerroken** [comment]
That will be perfect indeed! Thanks!

Van: Iván Chicano [mailto:notifications@github.com]
Verzonden: maandag 1 februari 2021 10:06
Aan: micado-eu/migrant_application
CC: Annelies Verroken; Author
Onderwerp: Re: [micado-eu/migrant_application] Description_image: caption text is necessary (#131)


The user will have in the image dialog a field to add a text alternative to the image such as this:
[chrome_VFUBEFTvnd]<https://user-images.githubusercontent.com/46921761/106436383-503b9c80-6474-11eb-8747-0d3b72d6695f.png>
When uploading the image only the image will be shown to normal readers, but it will be helpful to people that use screen readers (it would be read aloud like "image, text alternative to the image").
This field is not mandatory if the image has only decorative purposes or there is enough text context below the image.
About the level of WCAG I'm not entirely sure, maybe @gioppoluca<https://github.com/gioppoluca> knows more. I personally think that level AA is reasonable enough unless it was already specified to use AAA.

**Comment by ivchicano** [comment]
Already added, closing
